aesara/tensor/random/op.py

```diff
@@ -399,3 +399,43 @@ def grad(self, inputs, outputs):
 
     def R_op(self, inputs, eval_points):
         return [None for i in eval_points]
+
+
+class RandomMakerOp(Op):
+    def make_node(self, seed=None):
+        # np.inf is used as an representative of seed = None
+        if seed is None:
+            seed = np.inf
+        seed = as_tensor_variable(seed)
+        inputs = [seed]
+        outputs = [self.random_type()]
+        return Apply(self, inputs, outputs)
+
+    def perform(self, node, inputs, output_storage):
+        (seed,) = inputs
+        # np.inf is used as an representative of seed = None
+        if seed == np.inf:
+            seed = None
+        output_storage[0][0] = self.bitgen(seed=seed)
+
+
+class RandomStateMakerOp(RandomMakerOp):
+    random_type = RandomStateType()
+
+    def bitgen(self, seed):
+        return np.random.RandomState(seed=seed)
+
+
+random_state_maker = RandomStateMakerOp()
+
+
+class DefaultGeneratorMakerOp(RandomMakerOp):
+    random_type = RandomGeneratorType()
+
+    def bitgen(self, seed):
+        if seed.size == 1:
+            seed = int(seed)
+        return np.random.default_rng(seed=seed)
+
+
+default_rng_maker = DefaultGeneratorMakerOp()
```

Comment:
```suggestion
default_rng = DefaultGeneratorMakerOp()
```

This mirrors the NumPy interface and needs to be exposed via `aesara.tensor.random.__init__`.